agent/site.py

```diff
@@ -458,6 +459,12 @@ def tablewise_backup(self):
             data["tables"][table] = output
         return data
 
+    @step("Run Before Migrate Scripts")
+    def run_before_migrate_scripts(self, scripts: Dict[str, str]):
+        for app_name in scripts:
```

Comment:
```suggestion
        for script in scripts.values():
```

Reply:
valid, but not necessary ig :sweat_smile: